CMake/NmodlHelper.cmake

```diff
@@ -89,12 +89,18 @@ macro(nmodl_to_cpp_target)
 
   list(APPEND NMODL_${mod2c_KEY}_MODS "${mod2c_modname_}")
   if(CORENRN_ENABLE_NMODL AND NMODL_FOUND)
+    string(FIND "${NMODL_EXTRA_FLAGS_LIST}" "passes" pos)
```

Comment:
I think that `NMODL_EXTRA_FLAGS_LIST` should be `CORENRN_NMODL_FLAGS`. I cannot find where `NMODL_EXTRA_FLAGS_LIST` is set and it seems like it is not working properly.

Reply:
https://github.com/BlueBrain/CoreNeuron/blob/master/CMakeLists.txt#L187
:)

Reply:
Oops. Missed this.. :innocent: Sorry